packages/wonder-blocks-cell/src/components/__docs__/basic-cell.argtypes.js

```diff
@@ -0,0 +1,88 @@
+// @flow
```

Comment:
This file is used by Storybook so we can display the Props documentation properly.
<img width="1082" alt="Screen Shot 2021-12-21 at 6 45 29 PM" src="https://user-images.githubusercontent.com/843075/147011791-6ff6949b-ed8a-41c1-a22c-5702cbd8e520.png">